Makefile

```diff
@@ -9,6 +10,7 @@ JAVASCRIPTS_LOC=src/javascripts
 STYLESHEETS_LOC=src/stylesheets
 IMAGES_LOC=src/images
 REPORTS_FOLDER=reports
+PUGIN=parliament.uk-pugin
```

Comment:
Rename PUGIN to REPO, and move this a few lines down closer to GITHUB_API, REPO definitions?